Source/Controllers/DefaultAsyncMessagesCollectionViewDataSource.swift

```diff
@@ -37,14 +38,14 @@ class DefaultAsyncMessagesCollectionViewDataSource: NSObject, AsyncMessagesColle
     }
 
     //MARK: ASCollectionViewDataSource methods
```

Comment:
Nit: Change to `ASCollectionDataSource methods`.

Reply:
Done